functests/performance/rt-kernel.go

```diff
@@ -59,4 +60,20 @@ var _ = Describe("[performance]RT Kernel", func() {
 
 	})
 
+	It("[test_id:28526][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Non worker-cnf node should not have RT kernel installed", func() {
+
+		By("Skipping test if cluster does not have another available worker node")
+		nonRTWorkerNode, err := nodes.GetNonCNFNode(testclient.Client, "worker")
+		Expect(err).ToNot(HaveOccurred())
+
+		if nonRTWorkerNode == nil {
+			Skip("Skipping test because there are no additional non-cnf worker nodes")
+		}
+
+		fmt.Println(nonRTWorkerNode)
+		cmd := []string{"uname", "-a"}
+		Expect(execCommandOnWorker(cmd, nonRTWorkerNode)).To(ContainSubstring("Linux"), "Node should have Linux string")
```

Comment:
I see 
`functests/performance/rt-kernel.go:74:29: cannot use nonRTWorkerNode (type []"k8s.io/api/core/v1".Node) as type *"k8s.io/api/core/v1".Node in argument to execCommandOnWorker`

It looks like in nodes.go I am returning the wrong type.

Reply:
hey, as i see here you are returning list:
https://github.com/openshift-kni/performance-addon-operators/blob/b017c804fe9df5cd3d42ef2eb56dd45549a2add0/functests/utils/nodes/nodes.go#L37

and this one: as type _*"k8s.io/api/core/v1".Node_  means you need to add pointer.. so I think you need: &nonRTWorkerNode[0]